internal/bminventory/inventory.go

```diff
@@ -6477,26 +6479,58 @@ func isBaremetalBinaryFromAnotherReleaseImageRequired(cpuArchitecture, version s
 		featuresupport.IsFeatureAvailable(models.FeatureSupportLevelIDCLUSTERMANAGEDNETWORKING, version, swag.String(models.ClusterCPUArchitectureArm64))
 }
 
-// updateMonitoredOperators checks the content of the installer configuration and updates the list
-// of monitored operators accordingly. For example, if the installer configuration uses the
-// capabilities mechanism to disable the console then the console operator is removed from the list
-// of monitored operators.
-func (b *bareMetalInventory) updateMonitoredOperators(tx *gorm.DB, cluster *common.Cluster) error {
+// CheckClusterCapabilities sees if a cluster has the capabilities set to the
+// baselineCapability and contains all the capabilities listed in the includes list and excludes
+// any capability listed in the excludes list
+func (b *bareMetalInventory) CheckClusterCapabilities(cluster *common.Cluster, baselineCapability configv1.ClusterVersionCapabilitySet, includes, excludes []configv1.ClusterVersionCapability) (bool, error) {
```

Comment:
WDYT about a simpler approach: `HasBaseCapabilities(...)` and `HasAdditionalCapabilities(...)` instead of this single `CheckClusterCapabilities`? In this case we could move it to cluster package.

Reply:
Hmm I guess we could have the two split functions in the cluster package. I kept `CheckClusterCapabilities` here in inventory because of how it's imported into the BMAC, but I can try to move it to cluster

Reply:
Tried it out in this commit https://github.com/openshift/assisted-service/pull/7002/commits/510106ae0cff78ed491091919bb7e627dbf42d81
Wdyt?